### PR TITLE
feat: add ProfilesCreatedOverWeek component

### PR DESCRIPTION
**Pull request by geolessel**
This is the first multi-data-source component and is build mainly to
show how to create others in the future. It could be a useful
component on its own, I suppose...

Screen Shot 2020-05-05 at 3 35 25 PM

### CODE REVIEW

File: apps/dashboard_web/lib/dashboard_web/live/components/check_in_count.ex [hunk 1]
```diff
@@ -24,8 +24,9 @@ defmodule DashboardWeb.Components.CheckInCount do
       |> Map.put(:title, "Check-ins")
       |> Map.put(:product, :checkins)
       |> Map.put(:grid_width, 1)
-      |> Map.put(:data_key, :check_in_count)
       |> Map.put(:data_number, total_count)
+      |> Map.put(:timeframe, "This week")
+      |> Map.put(:comparison_number, 800)
```
Comment (jessejanderson) [inline apps/dashboard_web/lib/dashboard_web/live/components/check_in_count.ex:29]: Is this just a default comparison number? If so it should probably be `0`.

File: apps/dashboard_web/lib/dashboard_web/live/components/headcounts.ex [hunk 2]
```diff
@@ -24,8 +24,9 @@ defmodule DashboardWeb.Components.Headcounts do
       |> Map.put(:title, "Headcounts")
       |> Map.put(:product, :checkins)
       |> Map.put(:grid_width, 1)
-      |> Map.put(:data_key, :headcounts)
       |> Map.put(:data_number, assigns.total_count)
+      |> Map.put(:timeframe, "This week")
+      |> Map.put(:comparison_number, 800)
```
Comment (jessejanderson) [inline apps/dashboard_web/lib/dashboard_web/live/components/headcounts.ex:29]: Same as above.

File: apps/dashboard_web/lib/dashboard_web/live/components/check_in_count.ex (hunk 1 above)
Comment (geolessel) [reply apps/dashboard_web/lib/dashboard_web/live/components/check_in_count.ex:29]: It is just me transferring your `arbitrary_comparison_number` from here: https://github.com/geolessel/pco-dashboard/pull/28/files#diff-23618e65c70c9ee74c567191884882f3L17

File: apps/dashboard_web/lib/dashboard_web/live/components/headcounts.ex (hunk 2 above)
Comment (geolessel) [reply apps/dashboard_web/lib/dashboard_web/live/components/headcounts.ex:29]: It is just me transferring your `arbitrary_comparison_number` from here: https://github.com/geolessel/pco-dashboard/pull/28/files#diff-23618e65c70c9ee74c567191884882f3L17

**Pull request merged**